sretoolbox/container/image.py

```diff
@@ -127,7 +127,7 @@ def _raise_for_status(self, response, error_msg=None):
 
         msg += f'({response.status_code}) {response.reason}'
         content = json.loads(response.content)
-        for error in content['errors']:
+        for error in content.get('errors', []):
```

Comment:
Right! But could we use `None` as the default and only enter the for if `errors` is not `None`?